code/modules/organs/organ_external.dm

```diff
@@ -191,21 +192,25 @@
 					return
 			else
 				if(sharp || is_peg())
-					if(prob((5 * brute) * sharp)) //sharp things have a greater chance to sever based on how sharp they are
+					if(prob((5 * brute) * (sharp ? sharp : 1))) //sharp things have a greater chance to sever based on how sharp they are
 						droplimb(1)
 						return
-				else if(!sharp && brute > 15) //Massive blunt damage can result in limb explosion
-					if(prob((brute/7.5)**3)) //15 dmg - 8% chance, 22 dmg - 27%, 30 dmg - 64%, anything higher than ~35 is a guaranteed limbgib
-						explode()
-						return
-				else if(brute > 20 && prob(2 * brute)) //non-sharp hits with force greater than 20 can cause limbs to sever, too (smaller chance)
+				//15 dmg - 8% chance, 22 dmg - 27%, 30 dmg - 64%, anything higher than ~35 is a guaranteed limbgib
+				else if((brute > 15) && prob((brute/7.5)**3)) //Massive blunt damage can result in limb explosion
+					explode()
+					return
+				else if((brute > 20) && prob(2 * brute)) //non-sharp hits with force greater than 20 can cause limbs to sever, too (smaller chance)
 					droplimb(1)
 					return
 
-		else if((config.limbs_can_break && sharp == 100) || ((sharp >= 2) && (config.limbs_can_break && brute_dam + burn_dam >= (max_damage * config.organ_health_multiplier)/sharp))) //items of exceptional sharpness are capable of severing the limb below its damage threshold, the necessary threshold scaling inversely with sharpness
-			if(prob((5 * (brute * sharp)) * (sharp - 1))) //the same chance multiplier based on sharpness applies here as well
-				droplimb(1)
-				return
+		//items of exceptional sharpness and damage are capable of severing the limb below its damage threshold, the necessary threshold scaling inversely with sharpness
+		else if(sharp > 1 && (body_part != HEAD)) //Don't allow too-easy decapitations with this!
+			//Damage overflow, only the remaining damage after the reduction will be counted for the subsequent calculations
+			var/damage_overflow = get_health() + brute * sharp - max_damage * config.organ_health_multiplier
```

Comment:
thought this would make fountain pens and trader darts have a ~5% chance to cut off healthy limbs which would be fun but it turns out it doesn't
why the hell is `get_health()` called get health when it returns the total damage a limb has

Reply:
It's a 7 year old change that merged the "brute_dam + burn_dam" calculation into a proc. Coders, man.

Reply:
> It's a 7 year old change that merged the "brute_dam + burn_dam" calculation into a proc. Coders, man.

Is it time for a THANKS OLD CODERS tag?